includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -687,37 +803,81 @@ public function wc_setup_shipping_save() {
 	}
 
 	/**
-	 * Simple array of gateways to show in wizard.
+	 * https://stripe.com/global
+	 */
+	protected function is_stripe_supported_country( $country_code ) {
+		$stripe_supported_countries = array(
+			'AU' => __( 'Australia', 'woocommerce' ),
+			'AT' => __( 'Austria', 'woocommerce' ),
+			'BE' => __( 'Belgium', 'woocommerce' ),
+			'CA' => __( 'Canada', 'woocommerce' ),
+			'DK' => __( 'Denmark', 'woocommerce' ),
+			'FI' => __( 'Finland', 'woocommerce' ),
+			'FR' => __( 'France', 'woocommerce' ),
+			'DE' => __( 'Germany', 'woocommerce' ),
+			'HK' => __( 'Hong Kong', 'woocommerce' ),
+			'IE' => __( 'Ireland', 'woocommerce' ),
+			'JP' => __( 'Japan', 'woocommerce' ),
+			'LU' => __( 'Luxembourg', 'woocommerce' ),
+			'NL' => __( 'Netherlands', 'woocommerce' ),
+			'NZ' => __( 'New Zealand', 'woocommerce' ),
+			'NO' => __( 'Norway', 'woocommerce' ),
+			'SG' => __( 'Singapore', 'woocommerce' ),
+			'ES' => __( 'Spain', 'woocommerce' ),
+			'SE' => __( 'Sweden', 'woocommerce' ),
+			'CH' => __( 'Switzerland', 'woocommerce' ),
+			'GB' => __( 'United Kingdom (UK)', 'woocommerce' ),
+			'US' => __( 'United States (US)', 'woocommerce' ),
+		);
+
+		return array_key_exists( $country_code, $stripe_supported_countries );
+	}
+
+	/**
+	 * Simple array of "in cart" gateways to show in wizard.
 	 * @return array
 	 */
-	protected function get_wizard_payment_gateways() {
+	protected function get_wizard_in_cart_payment_gateways() {
+		$country    = WC()->countries->get_base_country();
+		$can_stripe = $this->is_stripe_supported_country( $country );
+
 		$gateways = array(
+			'stripe' => array(
+				'name'        => __( 'Stripe', 'woocommerce' ),
+				'image'       => WC()->plugin_url() . '/assets/images/stripe.png',
+				'description' => sprintf( __( '<p>Accept all major debit and credit cards from customers in 135+ countries on your site. <a href="%s" target="_blank">Learn more about Stripe</a>.</p><p class="payment-gateway-fee">Fee: 2.9%% + 30¢ per transaction</p>', 'woocommerce' ), 'https://wordpress.org/plugins/woocommerce-gateway-stripe/' ),
```

Comment:
Localisation should not contain HTML.

Reply:
I agree that this has too much HTML - are there examples in core of how to have inline links in translations?

Reply:
Fixed in d8b7c21.

Reply:
My motivation to keep the link in the HTML is proper RTL support. See this demo: https://codepen.io/anon/pen/JrKwvx